digitalocean/resource_digitalocean_kubernetes_node_pool.go

```diff
@@ -19,10 +19,13 @@ const digitaloceanKubernetesDefaultNodePoolTag = "terraform:default-node-pool"
 func resourceDigitalOceanKubernetesNodePool() *schema.Resource {
 
 	return &schema.Resource{
-		Create:        resourceDigitalOceanKubernetesNodePoolCreate,
-		Read:          resourceDigitalOceanKubernetesNodePoolRead,
-		Update:        resourceDigitalOceanKubernetesNodePoolUpdate,
-		Delete:        resourceDigitalOceanKubernetesNodePoolDelete,
+		Create: resourceDigitalOceanKubernetesNodePoolCreate,
+		Read:   resourceDigitalOceanKubernetesNodePoolRead,
+		Update: resourceDigitalOceanKubernetesNodePoolUpdate,
+		Delete: resourceDigitalOceanKubernetesNodePoolDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
@andrewsomething: If the user is importing just a node pool (to a Terrraform config that already has the cluster), then this function may need to call into the DO API to fill in the `cluster_id` attribute instead of being the pass-through. I can do that in a subsequent PR or in this PR. Thoughts?

Reply:
Never mind. I just went ahead and implemented that support, with working acceptance test.